.travis.yml

```diff
@@ -0,0 +1,33 @@
+language: bash
+services: docker
+
+branches:
```

Comment:
This section is making it so that the build isn't happening unless the branch is `develop`. I think that we should remove it and add a `before_deploy` phase that installs Terraform.